packages/nextjs/pages/address/[address].tsx

```diff
@@ -0,0 +1,74 @@
+import { useEffect, useState } from "react";
+import { useRouter } from "next/router";
+import { ethers } from "ethers";
+import { PaginationButton } from "~~/components/blockexplorer/PaginationButton";
+import { TransactionsTable } from "~~/components/blockexplorer/TransactionsTable";
+import { Address } from "~~/components/scaffold-eth";
+import { useFetchBlocks } from "~~/hooks/scaffold-eth";
+import { getTargetNetwork } from "~~/utils/scaffold-eth";
+
+const AddressPage = () => {
+  const router = useRouter();
+  const address = typeof router.query.address === "string" ? router.query.address : "";
+  const { blocks, transactionReceipts, currentPage, totalBlocks, setCurrentPage, isLoading } = useFetchBlocks();
+  const [balance, setBalance] = useState<string>("");
+  const targetNetwork = getTargetNetwork();
+
+  const provider = new ethers.providers.JsonRpcProvider("http://localhost:8545");
+
+  useEffect(() => {
+    const fetchBalance = async () => {
+      if (address) {
+        try {
+          const balance = await provider.getBalance(address);
+          setBalance(ethers.utils.formatEther(balance));
+        } catch (error) {
+          console.error("Failed to fetch balance: ", error);
+        }
+      }
+    };
+    fetchBalance();
+  }, [address]);
+
+  // Filter blocks that contain transactions related to the desired address
+  const filteredBlocks = blocks.filter(block =>
+    block.transactions.some(
+      tx => tx.from.toLowerCase() === address.toLowerCase() || tx.to?.toLowerCase() === address.toLowerCase(),
+    ),
+  );
+
+  return (
+    <div className="m-10 mb-20">
+      <div className="flex justify-start mb-5">
+        <button
+          className="btn bg-primary text-primary-content hover:bg-accent hover:text-accent-content shadow-md transition duration-500 ease-in-out transform hover:-translate-y-1 hover:scale-110"
```

Comment:
No need to go this fancy here :D 

We can do just `btn btn-primary` (or secondary?) to be consistent, and also it handles the hover state and everything.

Same with other buttons in this PR.